UPGRADE_GUIDE.md

```diff
@@ -2,6 +2,11 @@
 
 ## From `0.3.5` to `0.3.6`
 
+### Multithreaded connection removed
+
+The old `MultiThreadedSpatialOSConnectionHandler` has been removed and the method `SpatialOSConnectionHandlerBuilder.SetThreadingMode` has been removed to reflect this change.
+In a future release, threading will be implemented into the `SpatialOSConnectionHandler`.
```

Comment:
Does this mean threading is no longer handled? That's currently how I'm reading it - or is there some other underlying reason to remove this that users would understand?

Reply:
This means that currently, the mode to choose between a multi and single threaded connection handler has been removed. In the future, THE connection handler will be threaded through jobs as planned.

Reply:
I guess what I'm asking is what happens if someone was using the multi threaded handler? Are they no longer able to do that? Are there impacts onto users who were doing that before? We say it's going to be handled in the future, but what about the mean time?

Reply:
If someone was using this, they can no longer use it.
However, we've never publicly documented this feature outside of the api existing in the generated api reference docs.

Reply:
I think the only thing I'd suggest is removing "in a future release" unless we have a plan for it happening sometime soon. If it's part of the backlog and we can prioritize it, then that works, or if not let's get a ticket in for it if it's important. Otherwise maybe just remove that line.

Reply:
Removed the line